home-data/src/main/java/com/softserveinc/ita/homeproject/repository/PermissionRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.softserveinc.ita.homeproject.repository;
+
+import com.softserveinc.ita.homeproject.model.Permission;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface PermissionRepository extends CrudRepository<Permission, Long> {
```

Comment:
Just to be sure that you understand what `CrudRepository` does. Do we use everything from it? Wouldn't it be enough to use something smaller?